src/python/pants/backend/python/lint/pylint/rules_integration_test.py

```diff
@@ -188,7 +188,7 @@ def test_uses_correct_python_version(rule_runner: RuleRunner) -> None:
     assert "invalid syntax (<string>, line 2) (syntax-error)" in batched_py2_result.stdout
 
     assert batched_py3_result.exit_code == 0
-    assert batched_py3_result.partition_description == "['CPython<4,>=3.7']"
+    assert batched_py3_result.partition_description == "['CPython==3.7.*']"
```

Comment:
Pylint was dying on 3.8 with an `astroid` error due to the `ast` module removing `Str`.